scene/3d/sprite_3d.cpp

```diff
@@ -1184,7 +1176,10 @@ double AnimatedSprite3D::get_speed_scale() const {
 }
 
 Rect2 AnimatedSprite3D::get_item_rect() const {
-	if (!frames.is_valid() || !frames->has_animation(animation) || frame < 0 || frame >= frames->get_frame_count(animation)) {
+	if (frames.is_null() || !frames->has_animation(animation)) {
+		return Rect2(0, 0, 1, 1);
+	}
+	if (frame < 0 || frame >= frames->get_frame_count(animation)) {
```

Comment:
Why did you separate them?

Reply:
See https://github.com/godotengine/godot/pull/65148#discussion_r964949610.

At the end of the day, the compiler will likely just merge the two, so it's just a matter of readability.